infra/app/envs/dev/main.tf

```diff
@@ -1,5 +1,6 @@
 data "aws_caller_identity" "current" {}
 data "aws_region" "current" {}
+data "aws_lb" "current" {}
```

Comment:
this doesn't look right, how does this know which load balancer to select? unlike caller identity and region, there could be multiple load balancers. we should instead get the load balancer from module.app